test/incrParse/funcs.swift

```diff
@@ -16,11 +16,11 @@ class InvalidFuncDecls {
   func parensAdded<<ADD_FUNC_PARENS<|||()>>> {
   }
 
-  func openingBraceAdded() <<ADD_OPENING_BRACE|||{>>>
+  func openingBraceAdded() <<ADD_OPENING_BRACE<|||{>>>
 
   func closingBraceAdded() {
 
-  <<ADD_ClOSING_BRACE|||}>>>
+  <<ADD_ClOSING_BRACE<|||}>>>
```

Comment:
@akyrtzi looks like this was the typo that caused this test to fail.

Reply:
Good catch!